tests/FockSpace/FockSpace_test.cpp

```diff
@@ -25,12 +25,18 @@
 #include "Operator/SecondQuantized/SQHamiltonian.hpp"
 
 
+/**
+ *  Test the Fock space constructor
+ */
```

Comment:
Thanks for all of these changes: I know they're a pain, but this really improves the code!